js/NavMainComponent.js

```diff
@@ -74,8 +112,33 @@ NavMainComponent.prototype.switchSecondaryNav = function(target) {
     component.closeSubNavs();
 
     component.$html.find('.nav-list.is-active').removeClass('is-active');
-    component.$html.find('[data-nav="' + target + '"]')
-        .addClass('is-active');
+    component.$html.find('[data-nav="' + target + '"]').addClass('is-active');
+};
+
+NavMainComponent.prototype.switchTertiartyNav = function(target) {
+
+    var component = this;
+
+    component.$html.find('.navTertiary .nav-link').removeClass('is-active');
+
+    if (component.$html.find('[data-nav="' + target + '"]').length < 1) {
+        component.closeNavs();
+    }
+
+    component.$html.find('.navTertiary .is-active').removeClass('is-active');
```

Comment:
Actually .navTertiary is not used. The correct one is .nav-tertiary. Refactored all references of html.find to their relevant vars from the top of the file.